ESP32/DIY-Flow-Bench/hardware.cpp

```diff
@@ -56,14 +56,27 @@ Hardware::Hardware() {
 void Hardware::configurePins () {
  
   // Inputs
+  #ifdef VCC_3V3_PIN
   pinMode(VCC_3V3_PIN, INPUT); 
-  pinMode(VCC_5V_PIN, INPUT); 
```

Comment:
Will this need a #ifdef VCC_5V_PIN

Reply:
The 5v pin will always be used in a MAF system, but possibly not in a differential system. I suggest it is added.

Reply:
OK 👍